src/malli/generator.cljc

```diff
@@ -454,6 +454,20 @@
                         {:infinite? (get props :gen/infinite? false)
                          :NaN? (get props :gen/NaN? false)})
                       (-min-max schema options))))
+(defmethod -schema-generator :float [schema options]
+  (let [max-float #?(:clj Float/MAX_VALUE :cljs (.-MAX_VALUE js/Number))
+        min-float #?(:clj Float/MIN_VALUE :cljs (.-MIN_VALUE js/Number))
+        ceil-max (fn [v] (if (nil? v) max-float (min max-float v)))
+        floor-min (fn [v] (if (nil? v) min-float (max min-float v)))
+        props (m/properties schema options)
+        min-max-props (-min-max schema options)
+        infinite? (get props :gen/infinite? false)
+        NaN? (get props :gen/NaN? false)]
+    (gen/double* (merge {:infinite? infinite?
+                         :NaN? NaN?}
+                        (when (or (not infinite?) NaN?)
+                          {:min (floor-min (:min min-max-props))
+                           :max (ceil-max (:max min-max-props))})))))
```

Comment:
Here I was unsure how to exactly generate a float or a instance of `Float`, but I attempted to constrain the generated values to something that could be coerced into a `Float` instance.

If someone knows how/where to coerce the generated value into a Float instance then please let me know 🙏

Reply:
Call `(gen/fmap float (gen/double* ...))`.

Reply:
Although that doesn't work for `##Inf` and `##-Inf`. Might not make sense to allow them in a float generator, at least on the jvm.